### ISSUE TITLE
Add Indonesian translation

### ISSUE THREAD

**Issue by rozakbuhari** [single-spa/single-spa.js.org#428]
Hi @joeldenning, looks like there is no Indonesian translation yet for the website, would you mind setting up a repo at `id.single-spa.js.org`? I would love to contribute to it. Thank you!

**Comment by internettrans** [comment]
Hi @rozakbuhari, do you know anyone else who can help with the translation? We prefer having at least two people helping with the translation.

For reference, here was the issue for adding the Russian translation: https://github.com/single-spa/single-spa.js.org/issues/369

Here's the checklist for what needs to be done. Once we confirm there are at least two people willing to help with the translation, I'll start doing them.

### Tasks for me

- [x] Decide on a language code to use. I agree that `id` is good.
- [x] Create a new repo called `id.single-spa.js.org`. **Manually** initialize it to be the same git tree as single-spa.js.org. Do not do this by forking the repo -- a fork has a separate git tree. You can do this by initializing to empty repo (no readme, no gitignore) and then adding that as a remote to your local copy of `single-spa.js.org`:
```sh
cd single-spa.js.org
git remote add id git@github.com:single-spa/id.single-spa.js.org.git
git push id master:master
```
- [x] Update docusaurus config to correct domain (example at https://github.com/single-spa/ru.single-spa.js.org/commit/b8097893831736e3920d8d22a528c4f2458cbfad)
- [x] Update CircleCI badge in readme to correct url (example at https://github.com/single-spa/ru.single-spa.js.org/commit/e6660e0cdd2afb7c340b5362240d8f2f15721f76)
- [x] Enable builds in CircleCI. Add the GITHUB_TOKEN env variable
- [x] Modify [`/website/static/CNAME`](https://github.com/single-spa/single-spa.js.org/blob/master/website/static/CNAME) in to be the correct domain. https://github.com/single-spa/id.single-spa.js.org/commit/7bd77eb4e50f607dbbb32faca64db052feb0ce3e
- [x] Force HTTPS in the repository settings in Github. This takes a day or so after the CNAME is set for the SSL cert to be set up.
- [x] Create a pull request to the js.org repo, adding the subdomain. Should be similar to this: https://github.com/js-org/js.org/pull/3747
- [x] Add Indonesian to [translations.js](https://github.com/single-spa/single-spa.js.org/blob/master/website/src/data/languages.js), so that it appears in the list at https://single-spa.js.org/languages

### Tasks for @rozakbuhari  

- [x] Find at least one other person besides @rozakbuhari  who can help with the translations.
- [x] [Watch the id.single-spa.js.org Github repository](https://docs.github.com/en/enterprise-server@2.20/github/receiving-notifications-about-activity-on-github/watching-and-unwatching-repositories) for all activity
- [ ] Translate the files `/website/versioned_docs/version-5.x` directory. Do not translate the `/docs` directory. Also, translate each of the pages in `/website/pages`

### Tasks for @guguji5

- [ ] Set up the bot to automatically commit to id.single-spa.js.org whenever a PR is merged into single-spa.js.org.

**Comment by rozakbuhari** [comment]
Thanks for the response @joeldenning, I'll look for someone to help soon

**Comment by hryer** [comment]
I'll help for the translation 🙋🏼‍♂️ if it's okay or what can I do for helping you guys?

**Comment by rozakbuhari** [comment]
@hryer Thank you for being interested!

**Comment by internettrans** [comment]
The repository is created at https://github.com/single-spa/id.single-spa.js.org. I have created https://github.com/js-org/js.org/pull/5717 to add the domain. Also created https://github.com/single-spa/single-spa.js.org/pull/430 to add the link.

@rozakbuhari @hryer, please do the following:

1. Accept the invitation to the Github repository. You can do this by going to https://github.com/single-spa and there is a banner at the top.
2. [Watch the id.single-spa.js.org for all activity](https://docs.github.com/en/enterprise-server@2.20/github/receiving-notifications-about-activity-on-github/watching-and-unwatching-repositories)
3. Translate the files /website/versioned_docs/version-5.x directory. Do not translate the /docs directory. Also, translate each of the pages in /website/pages

**Comment by rozakbuhari** [comment]
Done!. The invitation accepted!, waiting for the the subdomain to live.

**Comment by internettrans** [comment]
The js.org subdomain was merged. Just waiting on the HTTP certificate:

![image](https://user-images.githubusercontent.com/5524384/112702351-cfd45f00-8e58-11eb-965e-b55a4820c9f5.png)

**Comment by internettrans** [comment]
@rozakbuhari @hryer it is live! See https://id.single-spa.js.org and https://single-spa.js.org/languages. Also, feel free to join the `#translations` channel in [this Slack workspace](https://join.slack.com/t/single-spa/shared_invite/zt-mafdeybq-0v1aIm3KKaqyVCT2xeny3Q) and say hello there!
